modules/rpc/handler.go

```diff
@@ -57,6 +58,7 @@ func (f OnewayHandlerFunc) HandleOneway(ctx fx.Context, reqMeta yarpc.ReqMeta, b
 }
 
 // WrapUnary wraps the unary handler and returns implementation of thrift.UnaryHandler for yarpc calls
+// TODO(anup): Remove host parameter once updated yarpc plugin is imported in the repo
```

Comment:
Is there a task ID associated with this? In general we should strive to associate TODOs with tasks, although I know that can be burdensome